container/libcontainer/handler.go

```diff
@@ -297,12 +298,25 @@ func processStatsFromProcs(rootFs string, cgroupPath string, rootPid int) (info.
 				socketCount++
 			}
 		}
+		mountpointPath := path.Join(rootFs, "/proc", pid, "mountpoint")
+		fileReader, err := os.Open(mountpointPath)
```

Comment:
curious how expensive this is, i.e. calling /proc/pid/mountpoint for every single pid?